phpunit/experimental/class-wp-navigation-fallbacks-gutenberg-test.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Tests WP_Navigation_Fallbacks_Gutenberg
+ *
+ * @package WordPress
+ */
+
+/**
+ * Tests for the WP_Navigation_Fallbacks_Gutenberg class.
+ */
+class WP_Navigation_Fallbacks_Gutenberg_Test extends WP_UnitTestCase {
+
+	public function test_it_exists() {
+		$this->assertTrue( class_exists( 'WP_Navigation_Fallbacks_Gutenberg' ) );
+	}
+
+	public function test_should_return_a_default_fallback_navigation_menu_in_absence_of_other_fallbacks() {
+
+		$data = WP_Navigation_Fallbacks_Gutenberg::get_fallback_menu();
+
+		$this->assertInstanceOf( 'WP_Post', $data );
+
+		$this->assertEquals( 'wp_navigation', $data->post_type, 'Fallback menu type should be `wp_navigation`' );
```

Comment:
```suggestion
		$this->assertEquals( 'wp_navigation', $data->post_type, 'Fallback menu type should be `wp_navigation`' );
```

I'd suggest to use `assertSame` as it uses strict comparison (`===`) to compare values.

Reply:
✅